### PR TITLE
Feature/utils file

### PR DESCRIPTION
**Pull request by ShaunaMyers**
Create and use utils file

## Description
- Create Utils file
- Add cleaning and formatting functions to utils file
- Replace timer variables with timer and setTimer in state
- Refactor handleAddTip for clarity
- Add timer to propTypes

## Concerns

-  Any bugs still present? No

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ import { faBars } from '@fortawesome/free-solid-svg-icons';
 const App = () => {
   const [advice, setAdvice] = useState([]);
   const [error, setError] = useState('');
+  const [timer, setTimer] = useState('')
```
Comment (tylrs) [inline src/components/App/App.js:16]: Great! this should get rid of our warnings in the console about timer

File: src/components/App/App.js [hunk 2]
```diff
@@ -50,19 +50,19 @@ const App = () => {
         return tip
       })
       setAdvice(updatedAdvice)
-      setError('You have successfully rated this tip')
-      timer = setTimeout(() => setError(''), 5000)
+      setError('You successfully rated a card')
```
Comment (tylrs) [inline src/components/App/App.js]: This message should stay the same or the test will fail

File: src/components/TipCard/TipCard.js [hunk 3]
```diff
@@ -5,14 +5,13 @@ import Rating from 'react-rating';
...
   
-  const [cardId, setId] = useState('')
-
-  let timer;
+  const [cardId, setId] = useState('');
+  const [timer, setTimer] = useState('');
 
   useEffect(() => {
-    timer = setTimeout(() => setId(''), 4000)
+    setTimer(setTimeout(() => setId(''), 1000))
```
Comment (tylrs) [inline src/components/TipCard/TipCard.js]: Also this needs to stay 4000

**Review by tylrs — CHANGES_REQUESTED** [review]

File: src/components/App/App.js (hunk 2 above)
Comment (ShaunaMyers) [reply src/components/App/App.js]: Changes have been made and this is corrected now.

File: src/components/TipCard/TipCard.js (hunk 3 above)
Comment (ShaunaMyers) [reply src/components/TipCard/TipCard.js]: Corrected this to say 4000 again

**Review by tylrs — APPROVED** [review]

**Pull request merged**